### DISCUSSION TITLE
What does PuDB mean?

### DISCUSSION THREAD

**Discussion by ranelpadon** [inducer/pudb#513] [Q&A]
I'm just curious about the **PuDB** acronym. Or what does the **u** mean? AFAIK, `pdb` means `Python debugger`. Also, is calling  `PuDB` as `pudb` in blog articles acceptable?

I checked the [FAQ](https://documen.tician.de/pudb/misc.html#faq) as well, and I'm wondering if **u** stands for [urwid ](https://urwid.org/)console UI library? I think it would be a good idea to include the meaning of `PuDB` in the project page/docs.

PS: Thanks for this awesome, visual debugger which is my favorite one. :)

**Comment by inducer** [comment]
Heh, good question.

Initially, the U was clearly for Urwid. These days, the name makes me think of semolina pudding in little Tupperware tubs, which my wife and I call "pud".

So take your pick, I guess? :)

**Comment by Freed-Wu** [comment]
Perhaps we can have a recursive acronym like `PuDB's user-friendly debugger` :smile:

**Reply by ranelpadon** [reply]
That's a good idea! hehe
